eng/Versions.props

```diff
@@ -149,7 +149,7 @@
     <SQLitePCLRawbundle_greenVersion>2.0.4</SQLitePCLRawbundle_greenVersion>
     <MoqVersion>4.12.0</MoqVersion>
     <FsCheckVersion>2.14.3</FsCheckVersion>
-    <SdkVersionForWorkloadTesting>6.0.200-preview.22055.18</SdkVersionForWorkloadTesting>
+    <SdkVersionForWorkloadTesting>6.0.300-preview.22159.1</SdkVersionForWorkloadTesting>
```

Comment:
This seems to be causing the following build failure:
```
  dotnet-install: Attempting to download using primary link https://dotnetcli.azureedge.net/dotnet/Sdk/6.0.300-preview.22159.1/dotnet-sdk-6.0.300-preview.22159.1-linux-x64.tar.gz
curl : (22) The requested URL returned error : 404 [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
```

https://dev.azure.com/dnceng/public/_build/results?buildId=1654017&view=logs&j=3a9f831a-39fc-5a98-005d-1222ec2c60ca&t=43ac2bf6-9cb4-5da7-8a9d-b264068ff198&l=3594

Reply:
@radical please take a look